src/test/java/DateTest.java

```diff
@@ -0,0 +1,33 @@
+import com.w6.nlp.DateTimeParser;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class DateTest {
```

Comment:
Can we add test with real dates like "first july" "1st july", "1st july of 2015" and so on?